src/components/DateTimeField/index.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import { InputText } from '@edx/paragon';
+import { getDateString, getTimeString } from '../../utils/index';
+import FieldLabel from '../FieldLabel';
+import { DATE_FORMAT } from '../../data/constants';
+
+class DateTimeField extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      date: getDateString(this.props.input.value) || '',
+      time: getTimeString(this.props.input.value) || '',
+    };
+    this.concatDateTime = this.concatDateTime.bind(this);
+    this.updateDate = this.updateDate.bind(this);
+    this.updateTime = this.updateTime.bind(this);
+  }
+
+  concatDateTime(date, time) {
+    const datetime = moment.utc(date + time, 'YYYY/MM/DD HH:mm').format(DATE_FORMAT);
```

Comment:
nit: Would prefer to do string interpolation here just so it's more clear that this isn't addition and instead we're joining two strings.

Reply:
Also looks like you duplicated this method in utils, can delete this here and remove the context binding above too if you import that?

Reply:
Do you think it'd be appropriate to remove it from utils and only have this method in the DateTimeField component? I originally had concatDateTime in utils because it was being accessed by three separate files, now it isn't used at all outside of DateTimeField.